ably-em-http-request.gemspec

```diff
@@ -3,17 +3,16 @@ $:.push File.expand_path('../lib', __FILE__)
 require 'em-http/version'
 
 Gem::Specification.new do |s|
-  s.name        = 'em-http-request'
-  s.version     = EventMachine::HttpRequest::VERSION
+  s.name        = 'ably-em-http-request'
+  s.version     = EventMachine::AblyHttpRequest::HttpRequest::VERSION
 
   s.platform    = Gem::Platform::RUBY
   s.authors     = ["Ilya Grigorik"]
-  s.email       = ['ilya@igvita.com']
-  s.homepage    = 'http://github.com/igrigorik/em-http-request'
+  s.email       = ['lawrence.forooghian@ably.com', 'lewis@lmars.net', 'matt@ably.io']
```

Comment:
The `email` field is [described as](https://guides.rubygems.org/specification-reference/#email):

> A contact email address (or addresses) for this gem

This is our fork; we shouldn’t be telling people to get in contact with the original author since this fork is not his responsibility.

Reply:
> MIT license does ask to include copyright notice in the project. LICENSE file doesn't exist in the original project, but there's a line at the end of the README file

What are you suggesting that I do? I can add an MIT `LICENSE` file and add the [`license` field](https://guides.rubygems.org/specification-reference/#license=) here, how about that?

Reply:
Oh, sorry, didn't realise the `license` field was already there. I’ll add a `LICENSE` file.

Reply:
Added a `LICENSE`.